docs/sql-data-sources-parquet.md

```diff
@@ -295,18 +295,6 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
     </p>
   </td>
 </tr>
-<tr>
```

Comment:
The configuration is not only about Parquet. Since it is disabled, and to avoid confusing users, I think we can just remove the doc here.